circle.yml

```diff
@@ -0,0 +1,3 @@
+machine:
+  node:
+    version: 5.12.0
```

Comment:
sorry for being late to this party, but @amishas157 @bkowshik - is there a reason we are using ubuntu 12.04? We should only need to support ubuntu 14.04 at the earliest. 12.04 is now over 5 years old, and I believe coming out of its LTS support period.